test/extended/builds/run_fs_verification.go

```diff
@@ -35,7 +35,7 @@ spec:
           value: "10"
       imageOptimizationPolicy: SkipLayers
     type: Docker
-`, image.ShellImage())
+`, image.LimitedShellImage())
```

Comment:
Or as @nalind mentioned during scrum, the cli image is built off of 

`registry.ci.openshift.org/ocp/4.8:base`

There is not helper method yet in the image component that exposes that image, but we could either try to add one there, or just reference it directly here.

Reply:
But it gets complicated cause we really should use an image stream image reference, and there is not imagestream for the base image right now.

This gets into the barriers we may encounter that I alluded to wrt adding imagestreams to the openshift namespace.
Similarly getting `registry.ci.openshift.org/ocp/4.8:base` into https://github.com/openshift/origin/blob/master/test/extended/util/image/image.go#L15 could prove difficult.

IMO the cli image via `LimitedShellImage` is the path of least resistance